docs/source/getting_started/03_custom_nodes.md

```diff
@@ -55,24 +48,24 @@ nodes:
 - model.yolo
 - dabble.bbox_count
 - draw.bbox
-- draw.bbox_count
+- draw.legend
 - output.screen
 - custom_nodes.output.csv_writer
 ```
 
 PeekingDuck is designed for flexibility and coherence between in-built and custom nodes; you can design a pipeline which has both types of nodes.
 
-As PeekingDuck runs the pipeline sequentially, it is important to check if the nodes preceding your custom nodes provides the correct inputs to your node.
+As PeekingDuck runs the pipeline sequentially, it is important to check if the nodes preceding your custom nodes provides the correct inputs to your node. PeekingDuck will return an error if the sequence of nodes are incorrect.
 
 ## Step 3: Create Node Configs
 
 Create a custom node config under `src/<custom_folder_name>/configs/<node_type>/<node>.yml`. In this tutorial it will be `src/custom_nodes/configs/output/csv_writer.yml` to list configurations required by the system.
 
 ```yaml
-input: ["count"]                         # inputs required by the node
-output: ["end"]                          # outputs of the node
-period: 1                                # time interval (s) between logs
-filepath: 'results/stats.csv'            # output file location
+input: ["count"]                # inputs required by the node
+output: ["none"]                # outputs of the node.
+period: 1                       # time interval (s) between logs
+filepath: 'results/stats.csv'   # output file location
```

Comment:
the section just before step 4 should include:
In our case, it will be `period` and `filepath`.